cpanfile

```diff
@@ -0,0 +1,18 @@
+##################################################
+# WARNING
+# This file is autogenerated by tools/update-deps
+# from dependencies.yaml
+##################################################
+
+requires 'Storable', '>= 3.06';
```

Comment:
I suggest to link https://github.com/preaction/Log-Any/pull/105 in our file(s) so that we know it's only a workaround for an upstream issue